src/main/java/edu/harvard/iq/dataverse/authorization/providers/oauth2/AbstractOAuth2AuthenticationProvider.java

```diff
@@ -127,19 +127,20 @@ public OAuth2UserRecord getUserRecord(String code, @NotNull OAuth20Service servi
         throws IOException, OAuth2Exception, InterruptedException, ExecutionException {
         
         OAuth2AccessToken accessToken = service.getAccessToken(code);
-    
+        final String userEndpoint = getUserEndpoint(accessToken);
```

Comment:
This userEndpoint variable doesn't seem to be used so I assume it can be deleted.

Reply:
Hi @pdurbin 

The microsoft.json don't have changes. The firts version of microsoft.json is used in this case too.

In case of userEndpoint variable is probably a old implementation of OAuth2, We don't know if other autenticathon service use this.

Reply:
@Gerafp ok, thanks, I still think QA might have questions but we'll see.

Here's a screenshot from my IDE (Netbeans) of the unused variable. It looks like you could use it below (line 138) if you want. Or you could delete it. Or you could do nothing. It's just something I noticed.

![Screen Shot 2019-10-23 at 5 08 58 PM](https://user-images.githubusercontent.com/21006/67434338-00ef9680-f5b8-11e9-9c9c-e61d1eb410b4.png)